starter-code/css/main.css

```diff
@@ -6,3 +6,166 @@ Green: #00B172
 White: #FFF
 
 */
+@import url("https://fonts.googleapis.com/css?family=PT Sans");
+
+body {
+  font-family: "PT Sans";
+  border: 1px solid #f0f2f5;
+  margin: 10px;
+}
+
+h1 {
+  color: white;
+  font-size: 5.5em;
+}
+
+h3 {
+  color: white;
+  font-size: 2em;
+  font-weight: lighter;
+  text-align: center;
+}
+
+.what h4 {
+  font-family: PT Sans bold;
+  font-size: 35px;
+  font-weight: bolder;
+  color: rgb(8, 224, 141);
+  text-align: center;
+}
+
+.images {
+  width: 100px;
+  height: 100px;
+  display: flex;
+  padding-left: 150px;
+  padding-right: 100px;
+}
+
+.imagelogo {
+  width: 250px;
+  height: 80px;
+}
+
+.linking {
+  list-style: none;
+  font-size: 1.6em;
+  font-weight: lighter;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  width: 500px;
+}
+
+.navbar {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  height: 100px;
+  padding-top: 20px;
+  align-items: center;
+}
+
+header {
+  background-image: url(../images/landing.jpg);
+  background-position: 0 0;
+  background-repeat: no repeat;
+  background-size: cover;
+  height: 700px;
+  padding: 50px 0 0 0;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  flex-wrap: wrap;
+}
+
+.description h2 {
+  font-size: 3em;
+  font-weight: lighter;
+  margin-bottom: 1px;
+  border-bottom: 3px solid rgb(8, 224, 141);
+  padding-bottom: 2px;
+}
+
+.description {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.examples h4 {
+  font-family: PT Sans bold;
+  font-size: 35px;
+  font-weight: bolder;
+  color: white;
+  padding: 0;
+  margin-top: 5px;
+  margin-bottom: 0;
+}
+
+.examples h2 {
+  color: white;
+  font-size: 2.5em;
+  font-weight: lighter;
+  margin-bottom: 2px;
+  border-bottom: 3px solid white;
+  width: 500px;
+  padding-bottom: 0;
+  padding-top: 50px;
+  padding-left: 100px;
+}
+
+.what {
+  padding: 100px 200px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-evenly;
+  align-items: center;
+  height: 300px;
+  width: 1200px;
```

Comment:
Following up with the previous review, the width is very wide so it makes it look that your website is a bit disordered. So maybe try taking out body and the width of `.what`see how it looks! :)